lib/embiggen/shortener_list.rb

```diff
@@ -13,7 +13,7 @@ def initialize(domains)
     end
 
     def include?(uri)
-      domains.any? { |domain| uri.host =~ /\b#{domain}\z/i }
+      domains.include?(uri.host)
```

Comment:
The regular expression ensures that domains in the list are anchored to the end of the host and are separated by a word boundary, without it `foobar.com` would be matched if `bar.com` is in the list.